### PR TITLE
EOS-23125: Extend Node CLI Support to Node Manifest View

### PR DESCRIPTION
**Pull request by sradhanand-pati**
Signed-off-by: sradhanandpati <sradhanand.pati@seagate.com>

### CODE REVIEW

**Comment by cortx-admin** [comment]
Can one of the admins verify this patch?

File: lr-cli/cortx_setup/commands/resource/show.py [hunk 1]
```diff
@@ -92,9 +96,8 @@ def run(self, **kwargs):
             # resource_map_path =
             # ResourceDiscover.get_resource_map(resource_type =
             # kwargs['resource_type'])
-            resource_map_path = get_pillar_data(
-                'provisioner/common_config/resource_map_path')
-            resource_dict = self.parse_resource_file(resource_map_path)
+            resource_map_path = get_pillar_data(HEALTH_PATH)
```
Comment (nitin-seagate) [inline lr-cli/cortx_setup/commands/resource/show.py:99]: Change name to: `resource_path`

File: lr-cli/cortx_setup/commands/resource/show.py [hunk 2]
```diff
@@ -39,7 +43,7 @@ def resource_filter_status(self, resource_state, resource_map):
         else:
             return resource_map
 
-    def parse_resource_file(self, resource_map_path: str):
+    def parse_file(self, resource_map_path: str):
```
Comment (nitin-seagate) [inline lr-cli/cortx_setup/commands/resource/show.py:46]: Change name from `resource_map_path` to: `file_map_path`

**Review by nitin-seagate — CHANGES_REQUESTED** [review]

File: lr-cli/cortx_setup/commands/resource/show.py (hunk 2 above)
Comment (sradhanand-pati) [reply lr-cli/cortx_setup/commands/resource/show.py]: Done

File: lr-cli/cortx_setup/commands/resource/show.py (hunk 1 above)
Comment (sradhanand-pati) [reply lr-cli/cortx_setup/commands/resource/show.py]: Done

**Review by 83bhp — APPROVED** [review]

**Pull request merged**